react-big-calendar/index.d.ts

```diff
@@ -38,6 +38,8 @@ declare module 'react-big-calendar' {
         messages?: Object;
         timeslots?: number;
         defaultView?: string;
+        className?: string;
+        elementProps?: React.HTMLAttributes<HTMLElement>;
```

Comment:
good changes but please could you add test cases to both className and elementProps?